src/app/components/AddCustomNetwork/index.js

```diff
@@ -0,0 +1,2 @@
+import AddCustomNetwork from './AddCustomNetwork'
+export default AddCustomNetwork
```

Comment:
This could be condensed to `export { default as AddCustomNetwork } from './AddCustomNetwork'`

Reply:
sure